### ISSUE TITLE
docs.gitops.weave.works broken/unreachable

### ISSUE THREAD

**Issue by MTRNord** [weaveworks/weave-gitops#4197]
It seems like something expired on the webhost as its failing with an xml saying "The project to be billed is associated with a delinquent billing account." currently. Is that already known?

**Comment by ksemele** [comment]
Maybe we need to go to GitHub pages for Weave instead?
Who maintains this repo at this time?

**Comment by ehcpdeveloper** [comment]
isn't there any other doc repo?

**Comment by Sijoma** [comment]
This also affects the helm chart repo right -> oci://ghcr.io/weaveworks/charts?

**Comment by kingdonb** [comment]
No, it would not affect the ghcr.io charts, @Sijoma 

We've created a #flux-ecosystem channel on the CNCF Slack instance where this can be discussed, there are some competing interests and someone is likely to re-publish and re-host the Weave GitOps docs soon, (potentially even with their own new branding) but in the mean time it's still possible to read them in the repo, by perusing the "website" directory that contains the docusaurus sources:

https://github.com/weaveworks/weave-gitops/tree/main/website

I'll admit that it's a bit unfriendly as a user interface, but it is at least not MIA.

I think there is still a goal to avoid a fork, and we should try to make this place as hospitable as possible for vendors to avoid creating a bunch of disparate forks. But from a centralization perspective, without maintainers and without formal governance, it will be hard for individuals to contribute to the Weave GitOps repos without forking them.

Anyway @Sijoma what you said is true about the legacy Helm chart repository, if users were using the http-based Helm chart repository (legacy Helm repository) then it is currently out of service.

OCI charts should all still be available (but I have no idea how many of published charts were actually hosted in an OCI repo.)

I have heard from a few different people they are getting good results from a `HelmRelease` backed by a `GitRepository` who have been testing WGE on their own, without rebuilding everything, for what that's worth! I won't recommend that as a long-term solution, (someone needs to publish and sign their own releases, make themselves known in #flux-ecosystem)

**Comment by kingdonb** [comment]
Please direct people to the CNCF channel [#flux-ecosystem](https://cloud-native.slack.com/archives/C06T28YH5B2) where we'll share the link, and take your questions and complaints about this refurbished docs site, manually published

We created a temporary (ish?) home for the docs now, https://docs.microscaler.com

Please direct any and all questions about this to the #flux-ecosystem channel linked above. Please do not share this link widely outside of the thread, I cannot guarantee it will stay up in its current form, but am posting it now as you all are interested and it is responsive to the thread here. Thank you for any additional feedback that you can provide!

If there's anything broken in the docs link above, please also report it but to keep this thread on-topic maybe you can report those issues on the separate repo.

The repo that I hosted it from is here: https://github.com/microscaler/gitops-docs

Am interested in recruiting maintainers who would want to keep this forked repo online long-term. I do not know if this link will stay good, I have hosted it on a personal account for now as I am personally interested in connecting Flux community and ecosystem with interested vendors and maintainers and/or customers, and really want to see all of this fixed ASAP.

**Comment by Sijoma** [comment]
I started to get 404 errors from the oci repository inside Flux. Without any changes. That's why I thought it was related aswell.

**Comment by kingdonb** [comment]
Doesn't seem to be down, but this might not be the same repository that you referenced:

https://github.com/weaveworks/weave-gitops/pkgs/container/charts%2Fweave-gitops

Can you share the HelmRepository and HelmRelease that failed @Sijoma ?

**Comment by kingdonb** [comment]
For right now, it's good to see that is still online. I wouldn't expect any of our OCI repos on GHCR to go down

But whoever is maintaining Weave GitOps into the future will probably opt to publish into their own (re-branded) chart repo.
I'd hope also adopting cosign keyless verification ;)

I think it's true that `HelmRepository` of `type: oci` does not actually reconcile anything. So without the HelmRelease that goes with that `HelmRepository` it would be meaningless to try to test. Anyway you said that it is working for you again now.

**Comment by Sijoma** [comment]
> For right now, it's good to see that is still online. I wouldn't expect any of our OCI repos on GHCR to go down

Sorry, I meant the error message is gone from the `status` but it still is `NotReady`

<img width="964" alt="image" src="https://github.com/weaveworks/weave-gitops/assets/8970764/fef9a67c-84f2-4849-8e99-012ca5fe2e3d">

**Comment by kingdonb** [comment]
@Sijoma Right, that is a bug in Weave GitOps, it does not understand that HelmRepository of `type: oci` does not have any status or reconciling, and `type: legacy` does indeed reconcile. This is a mild idiosyncrasy that was added in a recent version of Flux, (it's not actually a problem with the HelmRepository. Just a side effect of this item I mentioned above):

> I think it's true that HelmRepository of `type: oci` does not actually reconcile anything.

**Comment by Sijoma** [comment]
I've marked most of my comments as off-topic to not de-rail the original issue.

**Comment by casibbald** [comment]
Please use:
https://docs.gitops.weaveworks.org/
